--- Compress-Zlib-1.03/examples/filtdef	Sun Nov 23 21:16:32 1997
+++ Compress-Zlib-1.03/examples.new/filtdef	Sat Oct  2 21:07:45 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Compress::Zlib ;
 
--- Compress-Zlib-1.03/examples/filtinf	Sun Nov 23 21:16:32 1997
+++ Compress-Zlib-1.03/examples.new/filtinf	Sat Oct  2 21:07:53 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Compress::Zlib ;
 
--- Compress-Zlib-1.03/examples/gzcat	Mon Apr 22 17:12:04 1996
+++ Compress-Zlib-1.03/examples.new/gzcat	Sat Oct  2 21:08:01 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Compress::Zlib ;
 
--- Compress-Zlib-1.03/examples/gzgrep	Fri Feb 23 18:05:48 1996
+++ Compress-Zlib-1.03/examples.new/gzgrep	Sat Oct  2 21:08:08 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Compress::Zlib ;
 
--- Compress-Zlib-1.03/examples/gzstream	Sun Nov 23 21:16:32 1997
+++ Compress-Zlib-1.03/examples.new/gzstream	Sat Oct  2 21:08:16 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Compress::Zlib ;
 
